backend/Origam.DA.Service/MetaModelUpgrade/UpdateScriptContainers/XsltDataPageScriptContainer.cs

```diff
@@ -0,0 +1,40 @@
+#region license
+/*
+Copyright 2005 - 2024 Advantage Solutions, s. r. o.
+
+This file is part of ORIGAM (http://www.origam.org).
+
+ORIGAM is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+ORIGAM is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with ORIGAM. If not, see <http://www.gnu.org/licenses/>.
+*/
+#endregion
+
+using System.Collections.Generic;
+using Origam.Schema.GuiModel;
+
+namespace Origam.DA.Service.MetaModelUpgrade.UpdateScriptContainers;
+
+public class XsltDataPageScriptContainer : UpgradeScriptContainer
+{
+        public override string FullTypeName { get; } 
+            = typeof(XsltDataPage).FullName;
+
+        public override string[] OldPropertyXmlNames { get; } = {};
+        public override List<string> OldFullTypeNames { get; } 
+            = new() {"Origam.Schema.GuiModel.XsltDataPage"};
+
+        public XsltDataPageScriptContainer() 
+        {
+            AddEmptyUpgrade("6.0.0", "6.1.0");
+        }
+}
```

Comment:
Standard indentation is 4 spaces. Here you have 8. Please fix that.

Reply:
Fixed.